elvers/__main__.py

```diff
@@ -72,9 +72,9 @@ def build_dirs(ep_dir, params):
 
     # if desired, user can also provide out_path, and all dirs will be built under there
     # outdirectory, build all directories relative to that path
-    out_path = params.get('out_path', ep_dir)
+    out_path = params.get('out_path', os.getcwd())
```

Comment:
umm, this is actually not going to do what you want either, this does the same thing as it did before. If you don't specify a prefix of `/` for `os.path.join()`, it will look in the current directory - which is what `os.getcwd()` returns.

What you should do instead is, put some of your paths into variables, like I do in [2019-snakemake-cli](https://github.com/charlesreid1/2019-snakemake-cli):

https://github.com/charlesreid1/2019-snakemake-cli/blob/master/cli/command.py#L14-L16

Reply:
Sorry, ignore this comment, it was left in the wrong spot

Reply:
just checked the build to be sure - looks like results are going to the right spot now: `/home/travis/build/dib-lab/eelpond/nema_out/` rather than within the py egg